addons/sale/models/sale.py

```diff
@@ -506,6 +506,33 @@ def action_view_invoice(self):
             action = {'type': 'ir.actions.act_window_close'}
         return action
 
+    @api.multi
```

Comment:
remove, it's useless - it's actually an @api.model but it's useless anyway for private methods since never called through rpc

Reply:
@dbo-odoo api.multi removed